apps/aecore/lib/aecore/tx/pool/worker.ex

```diff
@@ -18,6 +18,7 @@ defmodule Aecore.Tx.Pool.Worker do
 
   @type tx_pool :: map()
 
+  @spec start_link(any()) :: :ignore | {:error, any()} | {:ok, pid()}
```

Comment:
this looks better than the huge spec used in the other `start_link`

Reply:
I can remove all `start_link` specs if you think that is better without them. I really don't like it!

Reply:
thank you